public/admin/config.yml

```diff
@@ -72,11 +79,28 @@ image_config: &image_config
       choose_url: false
       required: false
       hint: "Use https://squoosh.app/ to compress images before uploading. Choose the WebP format and resize the image to 2000px width."
-    - name: "alt"
-      label: "Alt Text"
-      widget: "hidden"
-      required: false
-      hint: "Refer to the W3C guidelines on when to use alt text: https://www.w3.org/WAI/tutorials/images/decision-tree/. Preferably, upload the image to ChatGPT 4.0 and use the prompt: 'Check the W3C guidelines (https://www.w3.org/WAI/tutorials/images/decision-tree/) and generate appropriate alt text for this image.'"
+    - *image_alt_config
+
+event_sponsor_config: &event_sponsor_config
+  - name: "name"
+    label: "Name"
+    widget: "string"
+  - name: "website"
+    label: "website"
+    widget: "string"
+    required: false
+  - name: "image"
+    label: "Image"
+    widget: "object"
```

Comment:
Maybe a hint? "Although optional, the sponsor won't be displayed if no image is provided."

Reply:
Actually - Image should be required. 🙏🏼